src/Symfony/Component/Console/Application.php

```diff
@@ -1125,12 +1125,12 @@ public function setDefaultCommand($commandName, $isSingleCommand = false)
     /**
      * @internal
      */
-    public function isSingleCommand()
+    public function isSingleCommand(): bool
     {
         return $this->singleCommand;
     }
 
-    private function splitStringByWidth(string $string, int $width)
+    private function splitStringByWidth(string $string, int $width): array
```

Comment:
Theoretically, `str_split()` below can return `false` (looking at callers of the `splitStringByWidth()` that could happen when the terminal width is `5` or less. How do we deal with that?

Reply:
This is the only call that I've found. If `splitStringByWidth` really returned `false` we would run into an error anyway.

https://github.com/symfony/symfony/blob/a0bbae75140584e51237c39b90ced54fdd261016/src/Symfony/Component/Console/Application.php#L805

Reply:
Yes, that's the call I referred to. But you are right.That's already broken. So only the error message changes. We may still need to think about fixing it. That's out of the scope of this PR though.